test/support/api.ex

```diff
@@ -28,6 +28,13 @@ defmodule MongoosePush.Support.API do
     }
   end
 
+  def get(path) do
+    {:ok, %Response{status_code: status, headers: headers, body: body}} =
```

Comment:
The `post/2` function uses `HTTPoison.post!`, while here we use pattern match with `HTTPoison.get` (no bang!). I don't mind either approach, but it would be nice to keep those functions behavior the same.